### PR TITLE
CVSB-444

### PR DESCRIPTION
**Pull request by mgherceanu**

### CODE REVIEW

File: src/models/tests/test-type.model.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export interface TestTypeModel {
   additionalNotesRecorded: string | null;
   additionalCommentsForAbandon: string | null;
   defects: DefectDetailsModel[];
+  linkedIds: string[] | null;
```
Comment (vladddragomir) [inline src/models/tests/test-type.model.ts]: move this property below the FE line

File: src/pages/testing/test-creation/test-types-list/test-types-list.ts [hunk 2]
```diff
@@ -61,31 +63,76 @@ export class TestTypesListPage implements OnInit {
...
+      } else return;
     }
   }
 
   cancelTypes() {
     this.navCtrl.pop();
   }
+
+  testsAllowed(addedTestypes: TestTypeModel[]): string[][] {
+    let allowedIds = [['1', '3', '4', '7', '8', '10', '14', '15', '16', '23', '18', '19', '21', '22', '27', '28', '30', '31', '32', '33', '34', '36', '38', '39']];
```
Comment (vladddragomir) [inline src/pages/testing/test-creation/test-types-list/test-types-list.ts]: please create a file in assets/app-data/test-types-data and retrieve the data from there

File: src/pages/testing/test-creation/test-types-list/test-types-list.ts [hunk 3]
```diff
@@ -61,31 +63,76 @@ export class TestTypesListPage implements OnInit {
...
+  }
+
+  testAlreadyAdded(addedTestTypes: TestTypeModel[], testType: TestTypesReferenceDataModel): boolean {
+    for (let elem of addedTestTypes) {
+      if (elem.testTypeId === testType.id) return true
+    }
+    return false
+  }
+
+  intersection(allowedTestsArray: string[][]): string[] {
```
Comment (vladddragomir) [inline src/pages/testing/test-creation/test-types-list/test-types-list.ts]: please add this to commonFunctionsService

File: src/models/tests/test-type.model.ts (hunk 1 above)
Comment (mgherceanu) [reply src/models/tests/test-type.model.ts]: done

File: src/pages/testing/test-creation/test-types-list/test-types-list.ts (hunk 2 above)
Comment (mgherceanu) [reply src/pages/testing/test-creation/test-types-list/test-types-list.ts]: done

File: src/pages/testing/test-creation/test-types-list/test-types-list.ts (hunk 3 above)
Comment (mgherceanu) [reply src/pages/testing/test-creation/test-types-list/test-types-list.ts]: done

**Pull request merged**